ufersa/plataforma-sabia
# Add announcement banner to home page

## Summary

Resolves #1098 

## QA Steps

1. Open home page and see that banner is rendered
2. Change to mobile viewport and the banner must change to mobile version

## Checklist
